src/annotator/config/index.js

```diff
@@ -28,6 +29,7 @@ export default function configFrom(window_) {
     onLayoutChange: settings.hostPageSetting('onLayoutChange'),
     openSidebar: settings.hostPageSetting('openSidebar', {
```

Comment:
There may be other values we want to force as a boolean, but only this one was needed for this ticket... and mostly because we only assume things coming from via are a string.